### PR TITLE
Build: Windows VHD Build pipeline check Windows Defender Signature

### PR DESCRIPTION
**Pull request by ShiqianTao**
**What type of PR is this?**
/kind build

**What this PR does / why we need it**:
**Which issue(s) this PR fixes**:
Windows VHD build pipeline check Signature to avoid the VHD has wrong Signature 

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [x] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by AbelHu** [comment]
@ShiqianTao Add "build:" before the commit message and the PR title.

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -309,6 +309,11 @@ function Get-SystemDriveDiskInfo {
     }
 }
 
+function Get-DefenderPreferenceInfo {
+    Write-Log "Get preferences for the Windows Defender scans and updates"
+    Get-MpPreference | Write-Output
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:314]: Does Write-Log not work?

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 2]
```diff
@@ -309,6 +309,11 @@ function Get-SystemDriveDiskInfo {
     }
 }
 
+function Get-DefenderPreferenceInfo {
+    Write-Log "Get preferences for the Windows Defender scans and updates"
+    Write-Log (Get-MpPreference)
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:314]: Write-Log (Get-MpPreference)
```
2022-04-28T12:23:29.9980773+08:00: MSFT_MpPreference (ComputerID = "F8734AC6-8C47-C032-C700-2970BF5DB9CB")
```
Use below code
```
Write-Log "Get preferences for the Windows Defender scans and updates"
Get-MpPreference | Write-OutPut
```

Comment (AbelHu) [reply vhdbuilder/packer/configure-windows-vhd.ps1:314]: Or
```
Write-Log(Get-MpPreference | Format-List | Out-String)
```

**Review by AbelHu — APPROVED** [review]

File: vhdbuilder/packer/configure-windows-vhd.ps1 (hunk 2 above)
Comment (ShiqianTao) [reply vhdbuilder/packer/configure-windows-vhd.ps1]: Updated. Thanks for your guidance.

**Pull request merged**